mpt/impl/managesieve/core/src/main/resources/org/apache/james/managesieve/scripts/capability.test

```diff
@@ -38,8 +38,9 @@ S: "VERSION" "1.0"
 S: OK
 
 C: AUTHENTICATE "PLAIN"
-S: \+ ""
-C:  user password
+S: ""
+S: OK
+C: "user password"
```

Comment:
I am not fully sure that this code is compliant with eg Thunderbird extension

From my memories of SASL we either have

```
C: AUTHENTICATE "PLAIN" "AGFsaWNlAHNlY3JldA=="\r\n
S: OK "Logged in"
```

Or

```
C: AUTHENTICATE "PLAIN"\r\n
S: +\r\n
C: AGFsaWNlAHNlY3JldA==\r\n
S: OK "Logged in"\r\n
```

(with base64 being `\0LOGIN\0password`)

So seing not base64 encoded, qutoed password seems indeed weird to me...

From the above exemple the initial server response is `+ \r\n`